README.md

```diff
@@ -39,12 +32,45 @@ def application do
 end
 ```
 
+## If you'd like to mount VerkWeb on another Endpoint:
+
+
+```elixir
+defmodule MyApp.Endpoint do
+  use VerkWeb.Mount, path: "/verk"
+  ...
+end
+```
+
+That should be it! :)
```

Comment:
I mean, he's already going to write something to mount the endpoint so it's not zero config. The standalone app can be used without configuration.